src/containers/errors/ServerError/style.css

```diff
@@ -0,0 +1,57 @@
+.server-error {
+  text-align: center;
+  width: 100%;
+}
+
+.server-error__img {
+  width: 80%;
+  margin-top: 137px;
+}
+
+.server-error__msg {
+  width: 229px;
+  height: 66px;
+  margin: 30px auto 20px auto;
+  font-family: Open Sans;
+  font-style: normal;
```

Comment:
by default : 
   font-style : normal
   font-weight : normal;
 #### so you can omit it